lua/lspsaga/codeaction.lua

```diff
@@ -148,8 +151,9 @@ function act:send_code_action_request(main_buf, options, cb)
     self.action_tuples = {}
 
     for client_id, result in pairs(results) do
+			local name = vim.lsp.get_client_by_id(client_id).name
```

Comment:
these code in wrong pos. you should get the name when the option equals to true.